test/test_files/annotations.dart

```diff
@@ -56,6 +56,10 @@ class PublicAnnotationClass {
         child2 = const PublicAnnotationClass.withAnIntAsOne();
 }
 
+class ExtendedAnnotationClass extends PublicAnnotationClass {
+  const ExtendedAnnotationClass():super();
```

Comment:
What happens if we have a parameter that is forwarded to the super constructor? If that isn't supported we should throw an error calling it out explicitly I think.